packages/plugin-packages/mashroom-websocket/src/backend/WebSocketServer.js

```diff
@@ -15,8 +16,10 @@ import type {
 import type {
     MashroomWebSocketServer,
 } from '../../type-definitions/internal';
+import TemporaryFileStore from './webapp/tempStore';
 
 const CHECK_CONNECTIONS_INTERVAL_MS = 30 * 1000;
+const RECONNECT_INTERVAL_MS = 10 * 1000;
```

Comment:
It would be cool if this interval could be configured - e.g. with a property "reconnectTimeout". Because the 10s could be too long if you just need a remedy for proxies that periodically disconnect your websocket connection or too short if you really want to "survive" serious network troubles e.g. on mobile devices